butler-bot/bot.js

```diff
@@ -63,4 +68,46 @@ client.on('message', message => {
     .then(() => { logger.info('Successfully added the role') })
 })
 
+client.setInterval((servers) => {
+  const itMuslimServer = servers.get(IT_MUSLIM_SERVER_ID)
+
+  const incompleteMembers = itMuslimServer.members.filter((member) => {
+    return !member.roles.has(FULL_MEMBER_ROLE_ID)
+  })
+  /* console.log(incompleteMembers) */
+
+  const incompleteMembersName = incompleteMembers.map((item) => {
+    return item.user.username
+  })
+  /* console.log(incompleteMembersName) */
+
+  const membersJoinTime = incompleteMembers.map((member) => {
+    return member.joinedTimestamp
+  })
+  /* console.log(membersJoinTime) */
+
+  /* const greetingsChannel = client.channels.get(GREETINGS_CHANNEL_ID) */
+
+  membersJoinTime.forEach((item) => {
+    let timeNow = new Date()
+    let timeDifference = (timeNow.getTime() - item) / 1000 / 60 / 60 / 24
```

Comment:
Please, create a separate constant like
```js
const MILLISECONDS_IN_DAY = 1000 * 60 * 60 * 24
```
and use it in your code.
Also, time difference is a very vague name. Let's call it `daysInactive`.